code/Test_definitions/DeviceIdentifier_retrieve_identifier.feature

```diff
@@ -0,0 +1,141 @@
+@DeviceIdentifier_retrieve_identifier
+Feature: Camara Device Identifer API retrieve identifier
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in code/API_definitions/CAMARA Mobile Device Identifier API.yaml
+# Implementation indications:
+# * api_root: API root of the server URL
+#
+# Testing assets:
+# * a mobile device with IMEI IMEI1
+# * a mobile device with IMEI IMEISV2, public IPv4 address PUBLICIPV4ADDRESS and public port PUBLICPORT
+# * a SIM card SIMCARD1 by telco1 and phone number PHONENUMBER1
+# * a SIM card SIMCARD2 by telco2 and phone number PHONENUMBER2
+
+  Background: Common Device Identifier retrieve identifer setup
+    Given the resource "/retrieve-identifier/v0"  as  base url
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the request body is compliant with the schema Device
+    
+  @DeviceIdentifier_retrieve_identifier0_phoneNumber_does_not_match_schema
+  Scenario Outline: phoneNumber value does not comply with the schema
+    Given the request body property "$.phoneNumber" is set to: <phone_number_value>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+
+    Examples:
+      | phone_number_value |
+      | string_value       |
+      | 1234567890         |
+      | +12334foo22222     |
+      | +00012230304913849 |
+      | 123                |
+      | ++49565456787      |
+
+  @DeviceIdentifier_retrieve_identifier100_success
+  Scenario:  retrieve device identifer for phone number PHONENUMBER1, network connection and access token matches PHONENUMBER1
```

Comment:
Use of "Oxford comma" is always advised for lists that are part of a specification
```suggestion
  Scenario:  retrieve device identifier for phone number PHONENUMBER1, network connection, and access token matches PHONENUMBER1
```